packages/webpack/builder.d.ts

```
@@ -1,9 +1,11 @@
 declare module '#builder' {
   import type Webpack from 'webpack'
   import type MiniCssExtractPlugin from 'mini-css-extract-plugin'
+  import type TsCheckerPlugin from 'fork-ts-checker-webpack-plugin'
 
   export const webpack: typeof Webpack
   export const WebpackBarPlugin: typeof import('webpackbar').default
   export const MiniCssExtractPlugin: typeof MiniCssExtractPlugin
+  export const TsCheckerPlugin: typeof TsCheckerPlugin
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TsCheckerPlugin'. Consider to delete it or rename it.

'TsCheckerPlugin' is defined here:


(lint/suspicious/noRedeclare)